client/src/components/OptionsMenu/OptionsMenu.component.tsx

```diff
@@ -0,0 +1,274 @@
+                  setQueryState(tagname)
+                  accordionRef.current?.click()
+                }}
+              >
+                <Flex m="auto" w="100%" px={8}>
+                  <Text>{tagname}</Text>
+                  <Spacer />
+                  <Flex alignItems="center">
+                    <BiRightArrowAlt />
+                  </Flex>
+                </Flex>
+              </Flex>
+            )
+          })
+        : agencyShortNamesToShow.map((shortname) => {
+            return (
+              <Flex
+                h="72px"
+                w={hasTagsKey ? '100%' : { base: '87%', sm: '100%' }}
+                mx={hasTagsKey ? undefined : { base: 'auto', md: undefined }}
+                alignItems="center"
+                textAlign="left"
+                textStyle="h4"
+                boxShadow="base"
+                role="group"
+                borderTopWidth={
+                  hasTagsKey ? { base: '1px', sm: '0px' } : undefined
+                }
+                borderTopColor={hasTagsKey ? 'secondary.500' : undefined}
+                bg="secondary.700"
+                color="white"
+                _hover={{ bg: 'secondary.600', boxShadow: 'lg' }}
+                as={RouterLink}
+                key={shortname}
+                to={getRedirectURLAgency(shortname)}
+              >
+                <Flex m="auto" w="100%" px={8}>
+                  <Text>{shortname}</Text>
+                  <Spacer />
+                  <Flex alignItems="center">
+                    <BiRightArrowAlt />
+                  </Flex>
+                </Flex>
+              </Flex>
+            )
+          })}
+    </SimpleGrid>
+  )
+
+  return (
+    <Accordion allowMultiple allowToggle index={hasTagsKey ? undefined : [0]}>
+      <AccordionItem border="none">
+        <AccordionButton
+          ref={accordionRef}
+          px="0px"
+          py="0px"
+          pt={hasTagsKey ? '24px' : undefined}
+          pb={hasTagsKey ? '16px' : undefined}
+          shadow="md"
+          _expanded={
+            hasTagsKey
+              ? { shadow: 'none' }
+              : !agency
+              ? { color: 'primary.500' }
+              : undefined
+          }
+          _hover={{ bg: hasTagsKey ? 'secondary.600' : undefined }}
+          bg={hasTagsKey ? 'secondary.700' : 'secondary.800'}
+        >
+          <Flex
+            maxW="680px"
+            m="auto"
+            w="100%"
+            px={8}
+            textAlign="left"
+            role="group"
+          >
+            <Stack spacing={1}>
+              <Text
+                textStyle="subhead-3"
+                color="primary.400"
+                pt={hasTagsKey ? '8px' : undefined}
+                mt={hasTagsKey ? undefined : '36px'}
+              >
+                {agency
+                  ? hasTagsKey
+                    ? 'TOPIC'
+                    : 'EXPLORE A TOPIC'
+                  : 'AGENCIES'}
```

Comment:
For copy consistency - change to 'EXPLORE AN AGENCY'?

Reply:
Seems a bit awkward as a phrase - one doesn't really explore an agency unless actually visiting the office